package.json

```diff
@@ -14,6 +14,8 @@
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "eslint": "^6.8.0",
+    "husky": "^4.2.5",
+    "lint-staged": "^10.2.2",
```

Comment:
Curious to hear your thoughts behind this. We used husky in the past, and personally I wasn't a big fan of it. But wanna hear your perspective on it.

Reply:
Husky can be a pain in the ass but I find it mostly helping in any kind of project. Most of the issues such as these (https://github.com/dynamoose/dynamoose/pull/890) won't happen with husky anymore (unless somebody skips commit prechecks). 

My thought is to always let tooling take any burden out of the developer's mind. I don't like to think about styling and usually forget about it when I am too focused on some issues. Then I submit PR and instead of moving forward with it I need somebody to correct me on my styling issues, which is just a slowdown and kinda annoying one. So if I can prevent it with anything I will. The argument would be to format files on save (which I also like to do) but for this project, my editor for some reason doesn't format according to the eslint rules.

TL;DR: I like to have consistent peace of mind for all developers that before they create PR their changes follow code styling of the project without much effort from their side.

Reply:
@pfulop And how about `lint-staged`? Never heard of that library before.